### PR TITLE
Syntax/readme ebelliveau

### PR DESCRIPTION
**Pull request by ebelliveau**
Minor syntax and spelling stuff.  Clean markdown is like proof-of-effort for thesis documents ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
+In Bitcoin, an unspent transaction output can only be spent when a previously-committed script is computed as valid. Bitcoin script is exhaustive with several `opcodes` which can be constructed into smart contracts. The most basic script is a `pay-to-single public key hash ([P2PKH](https://bitcoin.stackexchange.com/questions/64733/what-is-p2pk-p2pkh-p2sh-p2wpkh-eli5))` - where the proof-of-knowledge of one private key with a cryptographic signature returns valid, and thus enables the transfer of bitcoin. Only with the knowledge of a non-scarce piece of information of the private key can the property rights of a scarce bitcoin be proven and enforced. Scripts can be constructed with `OP_CHECKMULTISIG`, where a coin can only be spent with x-of-y valid signatures. The Schnorr signature scheme can aggregate individual public keys and signatures for a more efficient and secure multisig implementation. Since a `UTXO` can only be spent within a threshold agreement between independent individuals, they have non-simulated shared ownership of scarce libre sound money.
```
Comment (MaxHillebrand) [inline README.md]: I use m-of-n throughout the paper...
This seems to be somewhat of a standard in the context of Bitcoin [both for script and musig].

**Comment by ebelliveau** [comment]
<3 -- I'll keep rifling through the other docs this evening as time permits.  Does the branch naming convention work for you?
